### DISCUSSION TITLE
checking any event base wakeup of stm32u5

### DISCUSSION THREAD

**Discussion by rahulvilaspatil-eaton** [zephyrproject-rtos/zephyr#54803] [Q&A]
Hi, 
I am exploring event base wakeup of stm32u5 mcu. is there any support in zephyr which could help me to wakeup mcu from sleep mode .  I had try for RTC but did not get any success on it.

**Comment by yashi** [comment]
There is a section "[Wakeup capability](https://docs.zephyrproject.org/latest/services/pm/device.html#wakeup-capability)" in the documentation.  Hope it helps.

**Comment by erwango** [comment]
For now, only UART and I2C can be defined as wakeup sources using device tree and Zephyr PM subsystem.
For other sources, you should use STM32Cube HAL/LL API to configure wakeup capabilities
